src/fragments/lib/geo/js/geofences.mdx

```diff
@@ -32,18 +32,18 @@ import maplibregl from "maplibre-gl";
 import "maplibre-gl/dist/maplibre-gl.css";
 
 async function initializeMap() {
-    const el = document.createElement("div");
-    el.setAttribute("id", "map");
-    document.body.appendChild(el);
+  const el = document.createElement("div");
+  el.setAttribute("id", "map");
+  document.body.appendChild(el);
 
-    const map = await createMap({
-        container: "map",
-        center: [-123.1187, 49.2819], // [Longitude, Latitude]
-        zoom: 11,
-    })
+  const map = await createMap({
+    container: "map",
+    center: [-123.1187, 49.2819], // [Longitude, Latitude]
+    zoom: 11,
+  });
 
-    const control = new AmplifyGeofenceControl({})
-    map.addControl(control);
+  const control = new AmplifyGeofenceControl({});
+  map.addControl(control);
```

Comment:
Are we going to create another PR for adding docs for the geofence control docs?

Reply:
Created here - https://github.com/aws-amplify/docs/pull/3911